mirrord/kube/src/resolved/cron_job.rs

```diff
@@ -0,0 +1,47 @@
+use std::{borrow::Cow, collections::BTreeMap};
+
+use k8s_openapi::api::batch::v1::CronJob;
+
+use super::ResolvedResource;
+use crate::{
+    api::runtime::RuntimeDataFromLabels,
+    error::{KubeApiError, Result},
+};
+
+impl RuntimeDataFromLabels for ResolvedResource<CronJob> {
+    type Resource = CronJob;
+
+    fn name(&self) -> Cow<str> {
+        self.resource
+            .metadata
+            .name
+            .as_ref()
+            .map(Cow::from)
+            .unwrap_or_default()
+    }
+
+    fn container(&self) -> Option<&str> {
+        self.container.as_deref()
+    }
+
+    async fn get_labels(resource: &Self::Resource) -> Result<BTreeMap<String, String>> {
```

Comment:
I suggest renaming this method to `get_selector_match_labels`, because `get_labels` sounds a lot like it returns the resource's labels (which are under metadata.labels), like [`ResourceExt::labels()`](https://docs.rs/kube/latest/kube/trait.ResourceExt.html#tymethod.labels) does.